packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -130,6 +130,13 @@ export interface EmrCreateClusterProps extends sfn.TaskStateBaseProps {
    */
   readonly securityConfiguration?: string;
 
+  /**
+   * Specifies the step concurrency level to allow multiple steps to run in parallel. Requires Amazon EMR release label >=emr-5.28.0
+   *
+   * @default - EMR selected default
```

Comment:
I think this should describe the behaviour that users can expect. I know many properties in the EMR service integration have this documentation (mentioned in #8598), but we should avoid proliferating.